cmd/doctor.go

```diff
@@ -22,18 +25,24 @@ import (
 // CmdDoctor represents the available doctor sub-command.
 var CmdDoctor = cli.Command{
 	Name:        "doctor",
-	Usage:       "Diagnose the problems",
-	Description: "A command to diagnose the problems of current gitea instance according the given configuration.",
+	Usage:       "Diagnose and repair problems",
+	Description: "A command to diagnose and repair problems of current gitea instance according the given configuration.",
```

Comment:
Does this command intend to actually _repair_ problems or just report them to the user?

Reply:
_Repairing_ was my intention... but I didn't get too far. 😅